plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -5463,14 +5435,15 @@ private VirtualMachineGuestOsIdentifier translateGuestOsIdentifier(String cpuArc
                             }
                         }
                     }
-                    vmResponseMap.put(name, new VmStatsEntry( NumberUtils.toDouble(memkb)*1024,NumberUtils.toDouble(guestMemusage)*1024,NumberUtils.toDouble(memlimit)*1024, maxCpuUsage, networkReadKBs, networkWriteKBs, NumberUtils.toInt(numberCPUs), "vm"));
+                    vmResponseMap.put(name, new VmStatsEntry( NumberUtils.toDouble(memkb)*1024,NumberUtils.toDouble(guestMemusage)*1024,NumberUtils.toDouble(memlimit)*1024,
```

Comment:
Magic numbers. Don't we have an utils class to convert between these units, KB, bytes, bits, MB, MiB, GiB, GB...?

Reply:
I agree but the change here is an improvement and everybody is very busy. let's save it for a rainy day.